public/assets/styles/product.css

```diff
@@ -0,0 +1,253 @@
+.item__container {
+    width: 70vw;
+    height: auto;
+    margin: auto;
+    display: flex;
+    flex-direction: row;
+}
+
+.item__img__container {
+    min-width: 600px;
+    max-width: 600px;
+    height: 600px;
+    margin: 2rem;
+    border-radius: 5px;
+    overflow: hidden;
+}
+
+#item__img__slider {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.carousel-inner {
+    width: 100%;
+    height: 100%;
+}
+
+.carousel-item {
+    width: 100%;
+    height: 100%;
+}
+
+.carousel-item img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.item__info__container {
+    margin: 2rem;
+    text-align: left;
+}
+
+.item__info__container h2 {
+    font-size: 34px;
+    font-weight: bold;
+    letter-spacing: 1px;
+    text-transform: uppercase;
+}
+
+.item__info__container h5 {
+    margin-top: 1rem;
+    font-style: italic;
+}
+
+.item__info__container h5 span {
+    font-style: normal;
+    font-weight: bold;
+}
+
+.item__detail__container {
+    margin: 1rem 0;
+    border-top: 2px solid black;
+    border-bottom: 2px solid black;
+    padding: 1rem 0;
+}
+
+.item__detail__container h4 {
+    font-weight: bold;
+    font-size: 22px;
+}
+
+.item__detail__container p {
+    margin-top: 0.5rem;
+    font-size: 16px;
+    text-align: justify;
+    border-radius: 5px;
+    transition: all 0.2s linear;
+}
+
+.item__btn__container button {
+    width: 100%;
+    padding: 1rem 0;
+    font-size: 20px;
+    border-radius: 5px;
+    transition: all 0.2s linear;
+}
+
+.addBtn {
+    background-color: rgb(28, 27, 27);
+    color: #f1f1f1;
```

Comment:
You're mixing between hex colors and rgb() colors. It's best to just keep with one style unless you have to specify the alpha.